bundle/tests/scorecard/kuttl/test-iqe-jobs-selenium/01-pods.yaml

```diff
@@ -0,0 +1,91 @@
+---
+apiVersion: cloud.redhat.com/v1alpha1
+kind: ClowdEnvironment
+metadata:
+  name: test-iqe-jobs-selenium
+spec:
+  targetNamespace: test-iqe-jobs-selenium
+  providers:
+    testing:
+      # Level of access this pod has in the namespace
+      k8sAccessLevel: edit
+      # gather configuraiton for the specified environment
+      configAccess: environment
+      iqe:
+        # Base image for iqe-tests
+        imageBase: "quay.io/psav/clowder-hello"
+        resources:
+          limits:
+            cpu: 2
+            memory: 600Mi
+          requests:
+            cpu: 200m
+            memory: 256Mi
+        ui:
+          selenium:
```

Comment:
does it mean that other UI sidecars can be specified? I mean why specify `selenium` under `ui`?

Reply:
I think this is why yes, so that we leave it open in the future for different kinds of UI sidecars

Reply:
Correct ... just some future-proofing here